contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,172 @@
+| Control  | 5,000    | 500         | ❌ 10%             |
+| Test     | 5,000    | 1,000       | ✅ 20%             |
+
+At first glance, the test variant seems to be the clear winner. However, breaking down the results into the desktop and mobile subgroups shows:
+
+| Device           | Variant          | Visitors | Conversions | Conversion Rate |
+|-----------|------------------|----------|-------------|-----------------|
+| 💻 Desktop   | Control   | 2,000    | 400         | ✅ 20%             |
+|           | Test      | 2,000    | 100         | ❌ 5%              |
+| 📱 Mobile    | Control   | 3,000    | 100         | ❌ 10%             |
+|           | Test      | 3,000    | 900         | ✅ 30%             |
+
+It's clear now that while the test variant performed better for mobile users, it actually decreased desktop conversions. However, we missed this insight when we combined these metrics! This phenomenon is known as [Simpson's paradox](https://medium.com/homeaway-tech-blog/simpsons-paradox-in-a-b-testing-93af7a2f3307).
+
+Depending on your app and experiment, here's a list of aggregate metrics you want to breakdown:
+* Device type, e.g., desktop or mobile, iOS or Android.
+* Geographic location.
+* Subscription or pricing tier.
+* Acquisition channel, e.g., organic search, paid ads, or referrals, 
+* User role or job function, e.g., manager, executive, or individual contributor.
+* User tenure
+* Business size, e.g., small, medium, or large.
+
+## 3. Conducting an experiment without a predetermined duration
+
+You may even fall victim to the "[peeking problem](https://gopractice.io/data/peeking-problem/)": when you check the intermediate results for statistical significance, make decisions based on them, and end your experiment too early. Without determining how long your experiment should run, you cannot differentiate between intermediate and final results.
+
+Alternatively, if you do not have enough statistical power (i.e., not enough users to obtain a significant result), you'll potentially waste weeks waiting for results. This is especially common in [group-targeted experiments](/blog/running-group-targeted-ab-tests).
+
+The solution is to use an A/B test running time calculator to determine if you have the required statistical power to run your experiment and for how long you should run your experiment. This is built into PostHog.
+
+![Setting up a new experiment in PostHog includes a recommended running time calculator](../images/blog/ab-testing-mistakes/minimum-acceptable-improvement.png)
+<Caption>Setting up a new experiment in PostHog includes a recommended running time calculator</Caption>
+
+## 4. Running an experiment without testing it first
+
+Sometimes we are so eager to get results from our experiments that we jump straight into running it with all our users. This may be okay if everything is set up correctly, but if you've made a mistake, you may be unable to rerun your experiment. Let's look at why this is:
```

Comment:
```suggestion
Sometimes we're so eager to get results from our experiments, we jump straight to running them with all our users. This may be okay if everything is set up correctly, but if you've made a mistake, you may be unable to rerun your experiment. Why? Let me explain.
```